cucumber/authenticators/features/authn_config.feature

```diff
@@ -0,0 +1,86 @@
+Feature: Authenticator configuration
+
+  Background:
+    Given I have user "no-authn-access"
+    And I have user "authn-reader"
+    And I have user "authn-writer"
+    And a policy:
+    """
+    - !policy
+      id: conjur/authn-ldap/test
+      body:
+      - !webservice
+
+      - !webservice
+        id: status
+
+      - !group readers
+
+      - !permit
+        role: !group readers
+        privileges: [ read ]
+        resource: !webservice
+
+      - !group writers
+
+      - !permit
+        role: !group writers
+        privileges: [ update ]
+        resource: !webservice
+
+    - !grant
+      role: !group conjur/authn-ldap/test/readers
+      member: !user authn-reader
+
+    - !grant
+      role: !group conjur/authn-ldap/test/writers
+      member: !user authn-writer
+    """
+
```

Comment:
is the authenticator enabled or disabled when it is created? can we add a scenario to test this?

Reply:
Added a test to confirm that an unconfigured authenticator is disabled.